.gitmodules

```diff
@@ -12,8 +12,8 @@
 	branch = master
 [submodule "Tools/sitl_gazebo"]
 	path = Tools/sitl_gazebo
-	url = https://github.com/PX4/sitl_gazebo.git
-	branch = master
+	url = https://github.com/DanielDuecker/sitl_gazebo.git
```

Comment:
Can you open a corresponding PX4/sitl_gazebo PR and change this url back? 
https://github.com/PX4/sitl_gazebo

Reply:
For sure, I will update the url

Here is the PR in sitl_gazebo https://github.com/PX4/sitl_gazebo/pull/402